frontend/frontend/src/components/Home.vue

```diff
@@ -87,6 +122,14 @@ export default {
 	flex-direction: column;
 	align-items: center;
 }
+.radioButtons {
+	display: flex;
```

Comment:
Could we make the radio button labels white here in the css? Can we also make the radio button borders white instead of grey? Personally,  think there's too little contrast with the blue background. Let me know what you think!